src/t8_forest/t8_forest.cxx

```diff
@@ -477,7 +479,7 @@ t8_forest_element_diam (t8_forest_t forest, t8_locidx_t ltreeid, const t8_elemen
   dist = 0;
   for (i = 0; i < num_corners; i++) {
     /* Compute coordinates of this corner */
-    t8_forest_element_coordinate (forest, ltreeid, element, i, coordinates);
+    t8_forest_element_coordinate_from_corner_number (forest, ltreeid, element, i, coordinates);
```

Comment:
Can you change the name of the iterator variable to something mor meaningfull? Also reduce the scope of that variable please.